features-connection.js

```diff
@@ -186,6 +192,30 @@ module.exports = {
         return lifeTime > 0 ? lifeTime : null;
     },
 
+    sendingDuration: function(client, peerConnectionLog) {
+        let sendingDuration = 0;
+        let prevTime = timestamp(peerConnectionLog[0]);
+        let prevSending = false;
+        peerConnectionLog.forEach(log => {
```

Comment:
so far i've done for loops. If we want to go forEach then i'd recommend destructuring it as
{type, value, time, timestamp}
(and we add timestamp in extract.js)